examples/EvaluationDataToApplicationInsights/README.md

```diff
@@ -46,23 +46,10 @@ These cookies are used to correlate telemetry from the browser with telemetry fr
 
 *The Javascript SDK is not required, but is useful for collecting browser telemetry and generating these cookies out of the box.*
 
-### Authenticated User ID
-In order to connect metrics for the user between multiple services, a Authenticated User Id needs to be emitted. When the application is loaded, a login is simulated by setting a "username" cookie to a random integer. Additionally, the "ai_user" and "ai_session" cookies are expired, to simulate a new browser.
+### Targeting Id
+In order to connect evaluation events with other metrics from the user, a targeting id needs to be emitted. This can be done multiple ways, but the recommended way is to define a telemetry initializer. This initializer allows the app to modify all telemetry going to Application Insights before it's sent. 
 
-To include the authenticated user id on metrics emitted from the Javascript SDK, this app adds the following to _Layout.cshtml:
-```html
-appInsights.setAuthenticatedUserContext(getCookie("username"));
-```
-
-To include it on metrics emitted from the ASP.NET SDK, this app uses a TelemetryInitializer named `MyTelemetryInitializer`:
-```csharp
-builder.Services.AddSingleton<ITelemetryInitializer, MyTelemetryInitializer>();
-```
-
-The initializer sets the Authenticated User on the context object for all telemetry emitted from the server:
-```csharp
-telemetry.Context.User.AuthenticatedUserId = username;
-```
+This example uses the provided `TargetingHttpContextMiddleware` and `TargetingTelemetryInitializer`. The middleware adds `TargetingId` (using the targeting context accessor) to the HTTP Context as a request comes in- while the initializer checks for the `TargetingId` on the HTTP Context, and if it exists, adds `TargetingId` to all outgoing Application Insights Telemetry.
```

Comment:
The '-' seems odd. Should it be 
... as a requests comes in, while ...
or add a whitespace between the word "in" and the hyphen.